classes/class-wc-connect-nux.php

```diff
@@ -156,7 +156,28 @@ public function register_order_page_labels_pointer( $pointers ) {
 			return $pointers;
 		}
 
-		public static function get_banner_type_to_display( $status = array() ) {
+		/**
+		 * Check that the current user is the owner of the Jetpack connection
+		 * - Only that person can accept the TOS
+		 *
+		 * @return bool
+		 */
+		public function can_accept_tos() {
+			// Developer case
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG ) {
```

Comment:
I excluded the `manage_options` check mainly because we're already requiring that the user be able to install/activate plugins *and* manage the WC store. That seems like enough to me.